docs/tuning-guide.md

```diff
@@ -133,8 +133,15 @@ Configuration key: [`spark.rapids.sql.concurrentGpuTasks`](configs.md#sql.concur
 Default value: `1`
 
 The RAPIDS Accelerator can further limit the number of tasks that are actively sharing the GPU.
-This is useful for avoiding GPU out of memory errors while still allowing full concurrency for the
-portions of the job that are not executing on the GPU.  Some queries benefit significantly from
+It does this using a semaphore. When metrics or documentation refers to the GPU semaphore it
+is referring to this.
+This restriction is useful for avoiding GPU out of memory errors while still allowing full
+concurrency for the
```

Comment:
Nit: the wrapping here looks odd if one is examining the .md file directly via a terminal rather than via github's rendering.

Reply:
Feel free to ignore my rant. I will make the change. Really? One of the reasons for mark down is that it hides this when rendered. I have a similar issue just a few lines below. That one is there mostly because I want the diff to be small for reviewing. If we care enough to do it here why not do it everywhere and forget about the diff?

Reply:
One of the main advantages of markdown over other doc formats is that is is pretty human-readable in a terminal without requiring rendering in a web-page or other tool, so I'd like to keep it looking nice for those that are looking at things offline.  It is just a nit, and I may be in the minority feeling this way about our markdown formatting.